packages/evolution-frontend/src/components/inputs/__tests__/__snapshots__/InputRadioNumber.test.tsx.snap

```diff
@@ -241,73 +301,94 @@ exports[`Render InputRadioNumber InputRadioNumber without "over max" option 1`]
 >
   <div
     className="survey-question__input-radio-input-container"
+    onClick={[Function]}
   >
     <div
       className="label-input-container"
     >
       <input
         checked={false}
         className="input-radio"
-        id="test_1"
-        name="inputChoice_test"
+        id="test_1__input-radio__1"
+        name="test_1"
         onChange={[Function]}
+        onClick={[Function]}
         type="radio"
-        value={1}
+        value="1"
```

Comment:
From what I see here, the value used to be a number and is now string. Does it have side-effects?